docs/install/index.md

```diff
@@ -1889,9 +1889,38 @@ MXNet supports the Debian based Raspbian ARM based operating system so you can r
 
 These instructions will walk through how to build MXNet for the Raspberry Pi and install the Python bindings for the library.
 
+You can do a dockerized cross compilation build on your local machine or a native build on-device
+
 The complete MXNet library and its requirements can take almost 200MB of RAM, and loading large models with the library can take over 1GB of RAM. Because of this, we recommend running MXNet on the Raspberry Pi 3 or an equivalent device that has more than 1 GB of RAM and a Secure Digital (SD) card that has at least 4 GB of free memory.
 
-**Install MXNet**
+**Cross compilation build (Experimental)**
+
+## Docker installation
```

Comment:
Test this. The install page's markdown functionality is broken due to how the selector works.

Reply:
Can you tell me how to test?